en/config/actuators.md

```diff
@@ -1,139 +1,484 @@
+These steps are covered in [Output Assignment and Configuration](#output-assignment-and-configuration).
 
 ## Actuator Testing
 
-<!-- Not documented: "Identify and assign motors" -->
- 
-When testing actuators, make sure that:
-- Motors spin at the "minimum thrust" position.
+The _Actuator Testing_ section in lower-right corner provides sliders that can be used to test (and determine) actuator and motor settings. 
+A slider is provided for each output defined in the [Actuator Outputs](#actuator-outputs) section.
+The slider example below shows the section for a typical VTOL Tiltrotor airframe.
 
-  The sliders snap into place at the lower end, and motors are turned off (disarmed).
-  The "minimum thrust" position is the next slider position, which commands the minimum thrust.
-  For PWM motors, adjust the minimum output value such that the motors spin at that slider position (not required for DShot).
-  :::note
-  VTOLs will automatically turn off motors pointing upwards during fixed-wing flight.
-  For Standard VTOLs these are the motors defined as multicopter motors.
-  For Tiltrotors these are the motors that have no associated tilt servo.
-  Tailsitters use all motors in fixed-wing flight.
-  :::
-- Servos move into the direction of the convention described above.
-  :::note
-  A trim value can be configured for control surfaces, which is also applied to the test slider.
-  :::
+![Actuator Testing Slider](../../assets/config/actuators/vtol_tiltrotor_sliders_example.png)
+
+The section has an **Enable Sliders** switch that must be toggled before sliders can be used.
+The sliders can power the motors/servos across their full range of motion, and "snap" to the disarmed and minimum positions.
+
+:::note
+After you toggle the **Enable sliders** switch, actuators/motors won't do anything until the corresponding slider is _moved_.
+This is a safety feature to prevent sudden motor movements after switch is enabled.
+:::
+
+Sliders can be used to verify the following:
+
+1. Actuators (Motors, Control surfaces, etc.) are assigned to the expected output.
+1. Motors don't spin when at the `disarmed` PWM output value
+1. Motors barely spin at the `minimum` PWM output value.
+1. Motors give **positive thrust** in the expected direction
+1. Control Surfaces are in the correct idle position for `disarmed` output value
+1. Control Surfaces move in the direction as defined in the [Control Surface Convention](#control-surface-direction-conceptual)
+1. Motor Tilt Servos are in the correct idle position for `disarmed` output value
+1. Motor Tilt Servos move in the direction as defined in the [Tilt Servo Convention](#motor-tilt-servos-conceptual)
+
+
+## Output Assignment and Configuration
+
+In the _Actuator Outputs_ section on the right panel, you can assign and configure outputs, optionally using the [Actuator Testing](#actuator-testing) sliders:
```

Comment:
```suggestion
Outputs are assigned to functions and configured in the [Actuator Outputs](#actuator-outputs) section, while the  [Actuator Testing](#actuator-testing) sliders are commonly used to determine appropriate configuration values to enter.
```

Reply:
Actuator Testing also is used for determining control surface movement direction, trim and so on, so maybe this change makes it too specific to one aspect of Actuator Testing?